assets/src/components/d3/d3ViewsNarrative.js

```diff
@@ -0,0 +1,57 @@
+import { isInRange, isOutOfRange } from '../../util/math'
+
+const createResourcesText = (resourceData, resourceType, gradeSelection, weekRange) => {
+  const unAccessedResources = resourceData.filter(resource => resource.self_access_count === 0)
+  const accessedResources = resourceData.filter(resource => resource.self_access_count === 1)
+  const narrativeTextResources = {}
+  narrativeTextResources.weekRange = `Resources accessed from week ${weekRange[0]} to ${weekRange[1]}.  `
+  narrativeTextResources.resourceType = `Selected resource type ${resourceType.length === 1 ? 'is' : 'are'} ${resourceType}.`
+  narrativeTextResources.gradeFilter = gradeSelection.toUpperCase() !== 'ALL' ? `Filtering on grades ${gradeSelection}.  ` : ''
+  narrativeTextResources.resourcesUnaccessCount = `You have not yet accessed ${unAccessedResources.length} of ${resourceData.length} resources.  `
+  narrativeTextResources.resourcesAccessCount = accessedResources.length !== 0 ? `. You have accessed ${accessedResources.length} of ${resourceData.length} resources.  ` : ''
+  narrativeTextResources.resourcesUnAccessList = unAccessedResources.map(x =>
+    resourceType.length === 1
+      ? `${x.resource_name.split('|')[1]} accessed ${x.total_percent}%`
+      : `${x.resource_name.split('|')[1]} of type ${x.resource_type} accessed ${x.total_percent}%`
+  )
```

Comment:
Okay, sorry, I see that you did make these changes in an earlier commit, but for some reason they are not being reflected in the code I pull down, or here in the cumulative changes. Can you please make sure they are there?